tests/js/server/dump/dump-db-properties-cluster-all.js

```diff
@@ -0,0 +1,85 @@
+/*jshint globalstrict:false, strict:false, maxlen : 4000 */
+/*global assertEqual, assertNotEqual, assertTrue, assertFalse */
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief tests for dump/reload
+///
+/// @file
+///
+/// DISCLAIMER
+///
+/// Copyright 2019 ArangoDB GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Jan Christoph Uhde
+/// @author Copyright 2019, ArangoDB GmbH, Cologne, Germany
+////////////////////////////////////////////////////////////////////////////////
+
+const internal = require("internal");
+const jsunity = require("jsunity");
+const db = internal.db;
+
+function testProperties(name, minReplicationFactor, replicationFactor, sharding){
+  db._useDatabase(name);
+  let props = db._properties();
+  assertEqual(replicationFactor, props.replicationFactor);
+  assertEqual(minReplicationFactor, props.minReplicationFactor);
+  assertEqual(sharding, props.sharding);
+}
+
+const doIt = testProperties;
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test suite
+////////////////////////////////////////////////////////////////////////////////
+
+function dumpTestSuite () {
+  'use strict';
+
+  return {
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief set up
+////////////////////////////////////////////////////////////////////////////////
+
+    setUp : function () {
+      db._useDatabase("_system");
+    },
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief tear down
+////////////////////////////////////////////////////////////////////////////////
+
+    tearDown : function () {
+    },
```

Comment:
we should also have a `db._useDatabase("_system");` in the tearDown, to make sure that any follow-up action (other tests etc.) is guaranteed to be executed in the _system database and not elsewhere.